src/main/java/frc/robot/systems/DriveFSMSystem.java

```diff
@@ -423,21 +485,48 @@ private void handleTeleOpState(TeleopInput input) {
 			currentRightPower));
 
 		// turning in place
-		// if (Math.abs(rightJoystickY) < Constants.TELEOP_MIN_MOVE_POWER) {
-		//     power = Functions.turnInPlace(rightJoystickY, steerAngle);
-		// }
+		if (Math.abs(rightJoystickY) < Constants.TELEOP_MIN_MOVE_POWER) {
+			power = Functions.turnInPlace(rightJoystickY, steerAngle);
+		}
 
 		leftPower = power.getLeftPower();
 		rightPower = power.getRightPower();
 
-		if (input.getTopPressed()) {
+		if (input.getHangarButton()) {
```

Comment:
If we're executing a completely different algorithm for calculating our power, maybe this should be its own state?

Reply:
Yea it should be, but we were lazy. Since this is a pretty large change, we'll look at it in another pull request.